requirements-dev.txt

```diff
@@ -16,7 +16,7 @@ pytest-django>=4.9.0
 pytest-fail-slow>=0.3.0
 pytest-json-report>=1.5.0
 pytest-rerunfailures>=15
-pytest-sentry>=0.3.0
+pytest-sentry>=0.3.0,<0.4.0
```

Comment:
this PR doesn't do anything and isn't necessary.  we don't install from this file -- only use it to generate the lock